conf/application.conf

```diff
@@ -165,6 +165,7 @@ urls {
   ggSignOutUrl = "http://localhost:9553/bas-gateway/sign-out-without-state?continue=http://localhost:8994"
   eccEscSubscribeUrl = "http://localhost:6750/customs-enrolment-services/esc/subscribe"
   userResearchUrl = "#"
+  exchangeRateToolUrl = "https://ec.europa.eu/info/funding-tenders/procedures-guidelines-tenders/information-contractors-and-beneficiaries/exchange-rate-inforeuro_en"
```

Comment:
Should be `exchangeRateToUrl`

Reply:
I read this as "The url to the exchange rate tool?" - which would make `exchangeRateToolUrl` correct. Priyanka can let us know.

Reply:
Hi Sam, yes you are correct . @carwynellis it's exchange rate tool we are redirecting to, so I think exchangeRateToolUrl is ok

Reply:
Haha! I need more coffee!